flink-runtime/src/main/java/org/apache/flink/runtime/executiongraph/ExecutionGraph.java

```diff
@@ -475,13 +474,12 @@ public void enableCheckpointing(
 			checkpointTimeout,
 			minPauseBetweenCheckpoints,
 			maxConcurrentCheckpoints,
-			externalizeSettings,
+			retentionPolicy,
 			tasksToTrigger,
 			tasksToWaitFor,
 			tasksToCommitTo,
 			checkpointIDCounter,
 			checkpointStore,
```

Comment:
Here is one example where the proximity between the classnames `CompletedCheckpointStore` and `CheckpointStorage` becomes obvious.